lib/Widgets/DynamicNotebook.vala

```diff
@@ -818,6 +839,11 @@ namespace Granite.Widgets {
             notebook.set_action_widget (add_button_box, Gtk.PackType.START);
             notebook.set_action_widget (restore_button, Gtk.PackType.END);
 
+            //  delay tabs resizing until cursor leaves tab-bar
+            //  tab_bar-area = DynamicNotebook-area - TabPageContainer-are
```

Comment:
Typo: are->area
But maybe better to explain the formula in `recalc_size` instead?